models/examples/specification/example-a-validation-error-missing-field/Response-FhirError.json

```diff
@@ -9,7 +9,7 @@
           {
             "system": "https://fhir.nhs.uk/R4/CodeSystem/Spine-ErrorOrWarningCode",
             "version": "1",
-            "code": "VALIDATION_ERROR",
+            "code": "MISSING_FIELD",
```

Comment:
do we wanna get anymore of these examples in?

Reply:
sure, i'll put examples in the Validation errors section?

Reply:
decided not to add examples as they would be repeating information in table w/o much illustrative purpose